Framework/DataHandling/test/LoadNGEMTest.h

```diff
@@ -129,6 +152,58 @@ class LoadNGEMTest : public CxxTest::TestSuite {
     TS_ASSERT_THROWS(alg.execute(), const std::runtime_error &);
   }
 
+  void test_init_fails_on_histo_no_tof() {
+    LoadNGEM alg;
+    TS_ASSERT_THROWS_NOTHING(alg.initialize());
+
+    TS_ASSERT(alg.isInitialized());
+
+    TS_ASSERT_THROWS_NOTHING(alg.setProperty("PreserveEvents", false));
+    TS_ASSERT_THROWS_NOTHING(alg.setProperty("Filename", getTestFilePath("GEM000005_00_000_short.edb")));
+    TS_ASSERT_THROWS_NOTHING(alg.setProperty("OutputWorkspace", "ws"));
+    TS_ASSERT_THROWS(alg.execute(), const std::runtime_error &);
```

Comment:
Can these TS_ASSERT_THROWS be TS_ASSERT_THROWS_EQUALS to check the error message is correct?
Example: https://github.com/mantidproject/mantid/blob/main/qt/widgets/common/test/ConvolutionFunctionModelTest.h#L38-L42